reverse-number.js

```diff
@@ -0,0 +1,12 @@
+// Napisz funkcję reverseNumber(), która odwróci podaną liczbę (1234 => 4321)
+
+function reverseNumber(number) {
+    const newNumber = number.split("").reverse().join('');
+    return newNumber;
```

Comment:
There's no need to first declare this variable and then return it in the next step. You can write the whole function in one line.